.env.example

```diff
@@ -8,6 +8,7 @@ IMAGES_AWS_BUCKET=coordinape
 IMAGE_DIR=assets/static/images/
 # separate allowed domains with commas
 SIWE_ALLOWED_DOMAINS=localhost:3000,
+DEV_SEED_PHRASE="test test test test test test test test test test test junk"
```

Comment:
left here, but hopefully https://github.com/coordinape/coordinape/pull/1225 is merged ahead of this and can be removed